src/ReactWrapper.jsx

```diff
@@ -722,11 +722,19 @@ class ReactWrapper {
    * @param {*} initialValue - the initial value
    * @returns {*}
    */
-  reduce(fn, initialValue) {
-    return this.getNodes().reduce(
-      (accum, n, i) => fn.call(this, accum, this.wrap(n), i),
-      initialValue,
-    );
+  reduce(fn) {
```

Comment:
I believe @ljharb is trying to preserve `function.length`, which should only count required arguments. I think elsewhere we just use a default value of `undefined`, which might be good here.

```js
reduce(fn, initialValue = undefined) {
```